fontforge/parsettf.c

```diff
@@ -4764,9 +4777,14 @@ return;
 		    4*segCount*sizeof(uint16);
 	    /* that's the amount of space left in the subtable and it must */
 	    /*  be filled with glyphIDs */
+	    int rlen = len;
```

Comment:
what's this used for?

Reply:
Keeping the actual value read from the file in case it's needed. I was planning to write an informative error message but forgot to do so.